reward/reward_distributor.go

```diff
@@ -185,6 +186,20 @@ func GetTotalTxFee(header *types.Header, rules params.Rules, pset *params.GovPar
 	return totalFee
 }
 
+func GetTotalTip(header *types.Header, txs []*types.Transaction, receipts []*types.Receipt, rules params.Rules) *big.Int {
```

Comment:
Can we modify the GetTotalTxFee instead? without adding the GetTotalTip.

Reply:
Applied as adviced: https://github.com/klaytn/klaytn/pull/2187/commits/b97d8febf8ee0345f03d2e706d10dedd85ed3290